command/agent/config_parse.go

```diff
@@ -14,6 +14,15 @@ import (
 	"github.com/hashicorp/nomad/nomad/structs/config"
 )
 
+// ParseConfigDefault returns the configuration base
+func ParseConfigDefault() *Config {
+	return &Config{
```

Comment:
Are there more than just these three that need to have defaults set correctly?

Reply:
These are the only three where the defaults are required before decoding, other defaults are set in `command.readConfig`